ws/Cargo.toml

```diff
@@ -19,4 +19,7 @@ log = "0.4"
 pin-utils = "0.1.0-alpha.4"
 serde = { version = "1.0.40", features = ["derive"] }
 serde_json = "1.0.40"
-soketto = { git = "https://github.com/paritytech/soketto", rev = "a09bfb69af5650f9294e0e21bdda707e7d124b2c" }
+soketto = "0.3.1"
+# todo: add TLS feature for these?
+webpki = "0.21"
```

Comment:
This is alphabetically ordered

Reply:
Yeah this was in case I decide to add a feature for it. Not sure it's necessary though